examples/use-cases/push_events_to_syslog/pull_events.py

```diff
@@ -0,0 +1,83 @@
+import automox_console_sdk as automox
+import os
+import time
+import socket
+import json
+
+from automox_console_sdk import EventsApi
+from getpass import getpass
+
+def getRecentEvents(event_id=None):
```

Comment:
For consistency with other scripts and pep8, would recommend we have methods named with lowercase and underscore separation:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables